scripts/paper/generate_stwo_phase30_source_bound_manifest_reuse_figure.py

```
@@ -0,0 +1,429 @@
+    rows = read_rows(args.input_tsv)
+    validate_rows(rows, source=args.input_tsv)
+    grouped = group_rows(rows)
+    measured, runs = timing_metadata(rows, fallback_runs=args.bench_runs)
+    svg = build_svg(grouped=grouped, measured=measured, runs=runs)
+    args.output_svg.write_text(svg, encoding="utf-8")
+
+    if args.output_png is not None:
+        try:
+            rasterize(args.output_svg, args.output_png)
+        except (FileNotFoundError, subprocess.CalledProcessError):
+            pass
```

Ruff (0.15.11)

[warning] Use `contextlib.suppress(FileNotFoundError, subprocess.CalledProcessError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(FileNotFoundError, subprocess.CalledProcessError): ...`

(SIM105)